src/odemis/acq/move.py

```diff
@@ -61,6 +61,9 @@
 SAFETY_MARGIN_5DOF = 100e-6  # m
 SAFETY_MARGIN_3DOF = 200e-6  # m
 
+# tolerance for the movement axes checks
+ATOL_MOVEMENT_ROTATION = 0.04  # rad ~2.5 deg
+ATOL_MOVEMENT_TRANSLATION = 0.1e-6 # 0.1 um
```

Comment:
Be aware we already have such constants in odemis.util.driver `ATOL_*_POS`.  They are quite different though... but I'm not sure there is a good reason for it. They are used by `isNearPosition()`, which indicates where two positions are similar.
I think you could probably use `isNearPosition(... {"rz"})` to replace `has_axis_move`. Possibly extend that function to accept an optional argument `atol`.
Alternatively, you'd need to extend a little bit the comment to indicate they should not be mixed up with the util.driver.ATOL_* constants used to

Reply:
ah, i didnt see isNearPosition, i've changed this to use it now, and added the optional tolerance arg (one for linear, one for rot that default to driver.ATOL_*_POS). I've removed has_axis_move

the reasons for the additional constants, was that the rotation tolerance was 0.5 deg which is too strict for the leica stage, needed around 2.5deg. So i added these just for determining the transform, so they aren't as strict